pylab/core/models.py

```diff
@@ -8,18 +8,27 @@
 
 
 class Project(models.Model):
-    created = CreationDateTimeField()
-    modified = ModificationDateTimeField()
-    author = models.ForeignKey(User)
-    slug = AutoSlugField(populate_from='title')
-    title = models.CharField(_("Title"), max_length=255)
-    description = models.TextField(_("Description"))
+	PROJECT_STATUS_CHOICES = (
+		('Proposed', _('Proposed')),
+		('In progress', _('In progress')), 
+		('Suspended', _('Suspended')), 
+		('Implemented', _('Implemented')),
+	)
```

Comment:
See [Django coding style](https://docs.djangoproject.com/en/1.8/internals/contributing/writing-code/coding-style/#model-style) as an example how choices should be defined.

Also if order is important in list of choices, then choice value should be number. In this case order is important, because first project is proposed, then it gets in progress status, then it can be suspended and the last is implemented. Using numbers, you will be able to filter projects by choice more flexibly, for example you could ask for all projects whose status is greater than proposed.